### DISCUSSION TITLE
How to extend user information?

### DISCUSSION THREAD

**Discussion by donPuerto** [supabase/supabase#1869] [General]
I want to know how to able to extend information? ie First name phone1, phone2 etc//

**Comment by soedirgo** [comment]
Hey @donPuerto, if you use a `public.users` table as shown [here](https://supabase.io/docs/guides/auth#create-a-publicusers-table) you can add columns for your extra information. See for example the Slack Clone quickstart in the SQL editor with `username` and `status` columns.

**Reply by donPuerto** [reply]
Do we have any script tutorial?

**Reply by soedirgo** [reply]
Hmm, I don't have anything about extending the `public.users` table handy, but you can try going through https://supabase.io/docs/guides/with-nextjs to get a sense of how it works (the `profiles` table is basically the `public.users` table).

**Reply by donPuerto** [reply]
There should be documentation on how to play scripts.

**Comment by hallee9000** [comment]
I want this feature too. Storing in a public table is ok but it's not that convenient. You have to make another request to get extra information after user logging in.
